src/main/java/uk/gov/hmcts/juror/api/moj/domain/FormCode.java

```diff
@@ -13,8 +13,8 @@ public enum FormCode {
     BI_EXCUSAL("5225C", PrintDataService::printExcusalLetter, IJurorStatus.EXCUSED),
     ENG_EXCUSALDENIED("5226", PrintDataService::printExcusalDeniedLetter, IJurorStatus.RESPONDED),
     BI_EXCUSALDENIED("5226C", PrintDataService::printExcusalDeniedLetter, IJurorStatus.RESPONDED),
-    ENG_SUMMONS("5221", PrintDataService::printSummonsLetter, IJurorStatus.SUMMONED),
-    BI_SUMMONS("5221C", PrintDataService::printSummonsLetter, IJurorStatus.SUMMONED),
+    ENG_SUMMONS("5221", PrintDataService::reprintSummonsLetter, IJurorStatus.SUMMONED),
+    BI_SUMMONS("5221C", PrintDataService::reprintSummonsLetter, IJurorStatus.SUMMONED),
```

Comment:
I don't believe these should be reprint as they are just a summons letter and have the ability to be reprinted.

Reply:
Reprinted letters have a different history code added then initial printing :)